### ISSUE TITLE
wk4 may_epic: wave1 MTL Facilitation Guide - Upgrade to Facilitator Prep to increase team engagement

### ISSUE THREAD

**Issue by TomRust** [lzim/teampsd#396]
---
Step 1: Click raw to copy the code text below. 
Step 2: Click the link to create a new issue related to the SEE and SAY guides https://github.com/lzim/teampsd/issues/new.
Step 3: Edit the details to create a report that will help us improve the SAY and SEE guides.
labels: bug
assignees: @lzim, @staceypark

---

**Describe the bug**
Add guidance to prep to improve team engagement
 - start a chat with each team member in their individual world
 - see who runs experiments in their individual worlds, and then acknowledge them in the next team session

**Comment by staceypark** [comment]
@lzim @branscombj @dlounsbu @dlkibbe 
Can you weigh in on @TomRust suggestion to investigate and reinforce individual world activity? Should we add this to the SAY guides starting with Session 07 (first individual world homework is at the end of Session 06).

**Comment by branscombj** [comment]
> @lzim @branscombj @dlounsbu @dlkibbe
> Can you weigh in on @TomRust suggestion to investigate and reinforce individual world activity? Should we add this to the SAY guides starting with Session 07 (first individual world homework is at the end of Session 06).

I think yes, we should encourage playing in IND worlds. We do that in Session 5 when coaching their initial login to the Sim UI, and we can reiterate it during end-of-session DOs throughout S05-10. And we have a way to track logins and runs from the back-end.

I'm doubtful about the value add for them or us of encouraging use of the Sim UI chat. Love to hear how others envision that going, especially how likely the TAS think it'll be that they'll do it.

**Comment by lzim** [comment]
@TomRust @staceypark @branscombj @dlounsbu @dlkibbe @jessfroe 

> start a chat with each team member in their individual world, see who runs experiments in their individual worlds, and then acknowledge them in the next team session

Yes, we should encourage the IND worlds in the Done/Do review. I believe we do for each session, correct?

Given that, I do think that it is a fine proposal to chat them in their individual world and see who responds...Tom and Jane, would you want to pilot it tomorrow for Friday and see how it works?

😸

**Comment by dlkibbe** [comment]
In addition to what has already been suggested, another option is to create a short email template that is automatically sent to each team lead just after session 5 that is short and supportive, i.e.
"You and your clinic team can review and play with your team's data in your own simulation world. Just log in to mtl.how/sim and choose [TEAM_ind]. Contact [MTL email] if you have any questions."

@TomRust @staceypark @branscombj @dlounsbu @dlkibbe @jessfroe

If we do start a chat with each team member in their individual world, we should be sure we mention that function explicitly when we review the help functions.

**Comment by staceypark** [comment]
@TomRust @staceypark @branscombj @dlounsbu @dlkibbe @jessfroe 
Just to be clear, is this something we also need to change in the documentation of the SAY guides?

**Comment by lzim** [comment]
Cross-ref in the **meta_detail** card of the **document_tracker** #601 

Closes this issue #396

**Comment by branscombj** [comment]
@staceypark @dlkibbe I do not see above and do not recall what we decided about facilitators coaching teams on use of sim UI chat. I see this issue is closed, but it is referenced in the Document Tracker meta- cards for the See and Say guides so I don't know if any changes are needed or not.

**Comment by lzim** [comment]
@dlkibbe @branscombj 

Decided at Workgroup Leads Meeting on 3/30/20 to have Debbie and Jane take a look at this and disposition this.
- MTL 2.0 has a different team chat icon in the nav bar.
- Also, we will likely switch from Adobe Connect to Microsoft Teams for our virtual team meetings.
- But, this thread also is about ideas for engagement between sessions...

**Comment by lzim** [comment]
Changed the assignees @branscombj and @dlkibbe as this was still assigned to Stacey and I, but we switched it to you two back in March (see above).

**Comment by branscombj** [comment]
Hi @dlkibbe ! I'm reviewing all the Issues Assigned to Me and see this old one that was reopened and assigned to us to review and disposition. The bottom line as I see it is for us to think about and/or check our guides for the following. I'm going to make them a checklist; we can close this Issue when we've checked them all off.
- [ ] Do we adequately cover in the facilitation guides how to, and encouraging team members to, play in their individual worlds in the sim from sessions 5-10?
- [ ] Do we/how do we want to encourage use of the chat in the simUI?
- [ ] Are there other steps we should incorporate into MTL Live to foster team members' engagement between sessions (in any of the phases, I suppose, not just S05-S10)?

**Comment by lzim** [comment]
Added a **dependency** for the check list above related to the _MTL Facilitation & Technical Manual_ @staceypark and @lijenn

**Comment by branscombj** [comment]
> @staceypark May be able to answer this through HQ discussion
> 
> Hi @dlkibbe ! I'm reviewing all the Issues Assigned to Me and see this old one that was reopened and assigned to us to review and disposition. The bottom line as I see it is for us to think about and/or check our guides for the following. I'm going to make them a checklist; we can close this Issue when we've checked them all off.
> 
> * [ ]  Do we adequately cover in the facilitation guides how to, and encouraging team members to, play in their individual worlds in the sim from sessions 5-10?
> * [ ]  Do we/how do we want to encourage use of the chat in the simUI?
> * [ ]  Are there other steps we should incorporate into MTL Live to foster team members' engagement between sessions (in any of the phases, I suppose, not just S05-S10)?

@dlkibbe Could we discuss and resolve these questions this week for the Facilitator Manual.

**Comment by branscombj** [comment]
@lijenn @staceypark There is definitely an update needed to the Say guide(s) and probably See and elsewhere related to removing mentions of Adobe Connect - See [s05 Say in mtl.how](https://github.com/lzim/mtl/blob/master/blue/session05/s05_facilitator/mtl_session05_say.md). I am not sure where/how I should be noting this but I wonder if it's a Linter issue - at least to ID all the AdobeConnect mentions so they can be edited - not sure there's a universal replace option, but maybe. Seems like ideally we wouldn't name specific platforms unless we absolutely have to, since there's possibilities of them changing or even being substituted on an individual team basis for some reason, etc.)

In addition, and an example can be found in the same document, we need to change language that says Explorer is required for data UI - right? Now we can use Edge or Chrome or other with that I think?

**Comment by lijenn** [comment]
Good catch Jane, I'll create an issue card for myself to test the universal replace option (find and replace linter) and look for instances of Explorer as well as other updates needed to reflect MTL 2.0.  

Yes you are correct for Explorer, you can use any browser for mtl.how/data now.

**Comment by branscombj** [comment]
> * [ ]  Do we adequately cover in the facilitation guides how to, and encouraging team members to, play in their individual worlds in the sim from sessions 5-10?

Here is what we cover about IND world and what we ask/suggest they can do in IND between sessions (s05-s10 DOs):

- In s05 Say, we do describe the Ind and Team worlds as we're first logging in; and the DO at the end is to log in to team world but we describe that "if they're ambitious" they can log in to their IND world and load data as done in s05. 
- There is a bit of a leap, then for the DO at the end of s06 to say log in to IND and refine the Q and H written during the session. They will need to have the data loaded and selected to do this; and we will need to tell them that they can't pull up and edit the Team Q/H but will have to retype or copy/paste it from the team world. 
- s07 Do is explicit about just logging in to the Team world and reviewing the bc. We should probably also explicitly tell them not to change anything there in response to the rhetorical questions in the DO Demo.
- s08 DO says to log in to Team world and "Copy, paste, and edit your Decisions text to begin developing your new Question and Hypothesis for Experiment 2" but doesn't say where to paste and do the editing. In IND world?? 
- s09 DO says "if you can, run a 3rd exp in your ind world"-  "To do this, log back into the SIM UI. In Experiments, select your Experiment 2 run from the list of saved experiments. Click GO and select the box to include text, to reload your prior QUESTION, HYPOTHESIS, FINDINGS and DECISIONS." BUT they won't have those saved, Team experiments in their IND worlds - or can they? If not, how will they get bc, exp1 and exp2 into their IND worlds?
- s10 DO is just to review and think about all the exps and doesn't say where, but would be Team.

It would be nice if they could be logged in to the IND and Team worlds simultaneously in separate browsers; then they could review and paste back and forth. But I don't think this is possible.





> * [ ]  Do we/how do we want to encourage use of the chat in the simUI?
> * [ ]  Are there other steps we should incorporate into MTL Live to foster team members' engagement between sessions (in any of the phases, I suppose, not just S05-S10)?

**Comment by branscombj** [comment]
> 
> * [ ]  Do we/how do we want to encourage use of the chat in the simUI?

In s05 Say we mention the name (Team) and function (chat) in going over the toolbar. We don't say anything about when or why they'd chat their team, who sees it, or anything like that.

In s06 Say we ask them to paste some "i" text into "the chat box," but we do not say this in the s06 Checklist so maybe we're not doing that per se anymore? 

Note the verbiage is "chat box" so maybe what we're talking about here is the MSTeams chat? (grr - Teams chat vs simUI Team = chat --- confusing!)

Maybe the way to deal with the s08 DO is to say copy/paste into **the simUI Team chat** and edit there? We could do the same for the s06 DO, and let them run a personal exp3 (s09 DO) in the Team world, maybe put their name in the Q box for easy ID?
This would make it POSSIBLE to do all the DOs without loading data in their IND world or ever using it, unless they really are go-getters and want to do so. It would also capture any completed homework thinking in the simUI Team chat and create action there.

**Comment by jamesmrollins** [comment]
@lijenn to update the 9 clicks at the top and update the title.  This needs to be completed before Session 5 (second week of May).
